app/routes/helper_function.py

```diff
@@ -0,0 +1,16 @@
+from flask import  jsonify, abort, make_response
+
+def get_one_obj_or_abort(cls, obj_id):
```

Comment:
This is a great helper function.  I would also suggest making helper functions which validates the request body.